application/packages/python/ftrs_data_layer/domain/healthcare_service.py

```diff
@@ -20,6 +22,12 @@ class Telecom(BaseModel):
     web: str | None
 
 
+class AgeRangeType(BaseModel):
```

Comment:
```suggestion
class AgeRangeType(BaseModel):
range:  list {from:Decimal,to:Decimal} ,
type:TimeUnit
```

I think this needs to list of range in terms of consecutive age and in terms of non-consecutive age then its a list of AgeRangeType model 

Example 
Consecutive Range 
AgeRangeType(
    range=[{"from": 0, "to": 3},{"from":3,"to":5} ],
    type=TimeUnit.DAY
)

Non-Consecutive Days 
[
    AgeRangeType(
        range=[{"from": Decimal("1"), "to": Decimal("3")}],
        type=TimeUnit.DAY
    ),
    AgeRangeType(
        range=[{"from": Decimal("7"), "to": Decimal("9")}],
        type=TimeUnit.DAY
    )
]
@mark-spiers1  -Please correct me if i'm wrong that is what my understanding from the ticket